src/test/integration/statusBar.test.ts

```diff
@@ -0,0 +1,32 @@
+import * as assert from 'assert';
+import * as vscode from 'vscode';
+import { expect } from 'chai';
+import { terraformStatus } from '../../extension';
+import { getDocUri, open } from '../helper';
+import { sleep } from '../../utils';
+
+suite('statusBar', () => {
+  teardown(async () => {
+    await vscode.commands.executeCommand('workbench.action.closeAllEditors');
+  });
+
+  test('should create a status bar item', () => {
+    assert.notStrictEqual(terraformStatus, undefined);
+  });
+
+  test('should create a status bar with the root module label', async () => {
+    const documentUri = getDocUri('sample.tf');
+    await open(documentUri);
+    await sleep(500);
```

Comment:
@jpogran Do you have any idea how to avoid using `sleep` here to wait for the status bar to update?

Reply:
Resolved by using the function and awaiting